packages/kit/src/exports/public.d.ts

```diff
@@ -1943,9 +1943,11 @@ export type RemoteForm<Input extends RemoteFormInput | void, Output> = {
 	/** The number of pending submissions */
 	get pending(): number;
 	/** The submitted values */
-	input: null | UnionToIntersection<FlattenInput<Input, ''>>;
+	input: null | Partial<UnionToIntersection<FlattenInput<Input, ''>>>;
```

Comment:
This one's a little trickier. It's either `{}` before submission or it contains all the fields `UnionToIntersection<FlattenInput<Input, ''>>` after submission, so not exactly a `Partial` where each property can be `undefined` independently.
Maybe it's worth using this utility type https://stackoverflow.com/questions/70459727/how-to-define-all-or-nothing-type
```ts
type AllOrNothing<T extends any> = T | Partial<Record<keyof T, never>>;
```